### DISCUSSION TITLE
Restriction on Deleting Screenshots

### DISCUSSION THREAD

**Discussion by bagu05** [cattr-app/server-application#295] [Q&A]
Hi Team,

It has been observed that employees are currently able to delete screenshots without any restrictions. We need to implement measures to block this functionality and restrict employees from deleting screenshots.

Please advise on how we can customize this restrictions.

**Comment by DanilShapilov** [comment]
Hi. If I remember correctly, they can delete intervals(time+screenshot), not just screenshot.

**Reply by bagu05** [reply]
Are there any methods to prevent users from deleting screenshots?
